linera-service/src/cli_wrappers/local_kubernetes_net.rs

```diff
@@ -148,12 +153,19 @@ impl LineraNetConfig for LocalKubernetesNetConfig {
     }
 }
 
+#[cfg(with_testing)]
+pub struct LocalKubernetesTestNet {
```

Comment:
I believe `SharedLocalKubernetesTestingNet` would be consistent with `SharedLocalKubernetesNetTestingConfig`. Perhaps we can drop `Testing` altogether

Reply:
Ok, renamed. Thank you.